app/services/region_backfill.rb

```diff
@@ -83,13 +92,14 @@ def find_region_type(name)
   def create_region_types
     logger.info msg: "create_region_types"
     unless dry_run?
-      root = RegionType.create! name: "Root", path: "Root"
-      org = RegionType.create! name: "Organization", parent: root
-      facility_group = RegionType.create! name: "FacilityGroup", parent: org
-      block = RegionType.create! name: "Block", parent: facility_group
-      _facility = RegionType.create! name: "Facility", parent: block
+      RegionType::HIERARCHY.reduce(nil) do |parent_region_type, region_type|
```

Comment:
Do you need to reduce with a `nil` here, or could you just use `each` ? I think things would be more clear if you special cased the Root region type explictly here.